README.md

```diff
@@ -19,10 +19,14 @@ Can be installed via `pip`: `pip install codeflare-sdk`
 
 We recommend using Python 3.8 for development.
 Install development specific dependencies:
-  `$ pip install poetry pytest pytest-mock coverage black==22.3.0`
+  `$ pip install pre-commit poetry pytest pytest-mock coverage black==22.3.0`
```

Comment:
Could we put these in a requirements-dev.txt file? and then just do `$ pip install -r requirements-dev.txt`?

Reply:
Done!